integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -298,7 +310,12 @@ private int readInternal(
       LOG.error("Failed to read {}: Cannot find fd {}", path, fd);
       return -ErrorCodes.EBADFD();
     }
-    return entry.getFileStream().read(buf, size, offset);
+    try {
+      return entry.getFileStream().read(buf, size, offset);
+    } catch (NotFoundRuntimeException e) {
+      LOG.error("Failed to read {}: File does not exist or is writing by other clients", path);
+      return -ErrorCodes.ENOENT();
```

Comment:
nit: feel like we are double logging here. The principle is that if the user can see the information on the client side, we can reduce the logging on the server side.

Reply:
FUSE operations return an error code as a final answer to the operating system (no detail information is given to the end user, that's why we log the error here and ask end user to check the fuse.log). The operating system transfers the error code to a pre-defined message.
e.g. ENOENT() -> File/Dir does not exists

Reply:
Sad that we may not be able to give end-user informative information https://docs.alluxio.io/os/user/stable/en/api/POSIX-API.html#inputoutput-error-and-fuse-logs